src/video-player.js

```diff
@@ -50,11 +53,10 @@ class VideoPlayer extends Utils.mixin(Eventable) {
   constructor(elem, initOptions, ready) {
     super();
 
-    this.nbCalls = 0;
-
     this.videoElement = getResolveVideoElement(elem);
 
     this.options = extractOptions(this.videoElement, initOptions);
+    this.isLiveStream = this.options.videojsOptions.type === 'live';
```

Comment:
Should we set this as part of player constructor? Imean, the initial source can be live but the same player can switch to other sources that are not.
[Here](https://github.com/cloudinary/cloudinary-video-player/blob/ME-19040-live-stream-error/src/video-player.js#L261) we do `source.resourceConfig().type === 'live'`, would that be better?

Reply:
good point, I added `isLiveStream` fn that is called every time to check current type 👍

Reply:
anyway its edge case that type of live will be changed but anyway good to handle it